linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/ansible/shared.yml

```
@@ -4,7 +4,7 @@
 # complexity = medium
 # disruption = medium
 - name: "Read list of world and group writable system executables"
-  ansible.builtin.command: "find /bin /usr/bin /usr/local/bin /sbin /usr/sbin /usr/local/sbin /usr/libexec -perm /022 -type f"
+  ansible.builtin.command: "find -L /bin /usr/bin /usr/local/bin /sbin /usr/sbin /usr/local/sbin /usr/libexec -perm /022 \( -type l -o -type f \)"
```

The CI fail looks serious:

```
 [ 23%] Built target generate-internal-rhel9-cpe-oval-unlinked.xml
Error when trying to load the stream: while parsing a quoted scalar
  in "<unicode string>", line 2, column 28
found unknown escape character
  in "<unicode string>", line 2, column 122
HINT: Ansible tasks names may cause expansion errors if not properly quoted.
HINT: Check for unquoted colons or other special symbols in the stream:
```

It's most likely caused by the backslashes, they need to be escaped:

```suggestion
  ansible.builtin.command: "find -L /bin /usr/bin /usr/local/bin /sbin /usr/sbin /usr/local/sbin /usr/libexec -perm /022 \\( -type l -o -type f \\)"
```